app/src/main/java/com/ivy/wallet/ui/edit/core/EditBottomSheet.kt

```diff
@@ -110,6 +113,13 @@ fun BoxWithConstraintsScope.EditBottomSheet(
                 shadowRadius = 24.dp
             )
             .background(IvyTheme.colors.pure, Shapes.rounded24Top)
+            .scrollable(
```

Comment:
Hi, thanks for contributing! Can you use our internal `.verticalSwipeListener()`?